backend/routes/events.js

```diff
@@ -11,10 +11,17 @@ const processInput = require('../helpers/processInput');
 const eventsQueries = require('../queries/events');
 
 // Get all events (past events are auto pushed to the back)
-router.get('/all', async (req, res, next) => {
+router.get('/all/', async (req, res, next) => {
     try {
-        let allEvents = await eventsQueries.getAllEvents();
-        res.json({
+        const vName = processInput(req.query.v_name, "softVC", "volunteer name", 60);
```

Comment:
To refer to my previous comment, we can do this:
const vName = processInput(req.query.v_name, "softVC", "volunteer name", 60).toLowerCase();
const topic = processInput(req.query.topic, "softVC", "event topic", 50).toLowerCase();
const instructor = processInput(req.query.instructor, "softVC", "event instructor", 100).toLowerCase();

Reply:
👍🏽